product-page.css

```diff
@@ -0,0 +1,97 @@
+.product-page-main {
+    font-size: 1.6rem;
+}
+
+.filter-column {
+    background-color: var(--white);
+    align-self: stretch;
+    gap: 1rem;
+    min-width: 300px;
+    width: 20vw;
+    padding: 3rem;
+    box-shadow: 0px 6px 16px 0px var(--shadow-gray);
+    line-height: 2;
+    align-items: stretch;
+   }
+
+.product-display-section {
+    width: 80vw;
+    height: 100vh;
+    padding: 10rem 4rem;
+    overflow-y: scroll;
+}
+
+.line-height-lg {
+    line-height: 4;
+}
+
+.line-height-s {
+    line-height: 1.2;
+}
+
+.main-space-between {
+    justify-content: space-between;
+}
+
+.cross-space-between {
+    align-items: space-between;
```

Comment:
align-items : space-between I don't think so we have any property like this . So make correction in it